saleor/graphql/webhook/tests/benchmark/fixtures.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+from .....app.models import App
+from .....webhook.event_types import WebhookEventAsyncType, WebhookEventSyncType
+from .....webhook.models import Webhook, WebhookEvent
+
+NUMBER_OF_WEBHOOKS_PER_APP = 6
+
+
+def _prepare_webhook_events(option, webhook):
+    webhook_events = []
+
+    if option == 0:
```

Comment:
Can't we somehow have a few different fixtures that always create a certain number of specific webhooks? I would keep the fixtures as simple as possible, without any additional logic with these options or modulo division.